tpcx_bb/cluster_configuration/cluster-startup.sh

```diff
@@ -42,20 +43,19 @@ export DASK_DISTRIBUTED__COMM__RETRY__DELAY__MAX="60s"
 
 
 # Setup scheduler
-if [ "$HOSTNAME" = $SCHEDULER ]; then
+if [ "$ROLE" = "SCHEDULER" ]; then
   if [ "$CLUSTER_MODE" = "NVLINK" ]; then
-     CUDA_VISIBLE_DEVICES='0' DASK_UCX__CUDA_COPY=True DASK_UCX__TCP=True DASK_UCX__NVLINK=True DASK_UCX__INFINIBAND=False DASK_UCX__RDMACM=False nohup dask-scheduler --dashboard-address 8787 --interface $INTERFACE --protocol ucx > $LOGDIR/scheduler.log 2>&1 &
+     CUDA_VISIBLE_DEVICES='0' DASK_UCX__CUDA_COPY=True DASK_UCX__TCP=True DASK_UCX__NVLINK=True DASK_UCX__INFINIBAND=False DASK_UCX__RDMACM=False nohup dask-scheduler --dashboard-address 8787 --protocol ucx --scheduler-file $SCHEDULER_FILE > $LOGDIR/scheduler.log 2>&1 &
   fi
   
   if [ "$CLUSTER_MODE" = "TCP" ]; then
-     CUDA_VISIBLE_DEVICES='0' nohup dask-scheduler --dashboard-address 8787 --interface $INTERFACE --protocol tcp > $LOGDIR/scheduler.log 2>&1 &
+     CUDA_VISIBLE_DEVICES='0' nohup dask-scheduler --dashboard-address 8787 --protocol tcp --scheduler-file $SCHEDULER_FILE > $LOGDIR/scheduler.log 2>&1 &
```

Comment:
Why did we get rid of `INTERFACE` here, I remember @ayushdg  seeing quite a bit of speed up at least on RAPLAB with this even with pure TCP.

Reply:
I believe that is correct for that cluster. My primary goal is to reduce the configuration complexity, but if we want to keep this parameterized we can certainly do so

Reply:
This PR generally removes the `INTERFACE` environment variable, but we still rely on `network_interface=os.environ.get("INTERFACE", "ib0")`. Does it make sense to re-add the option of specifying that variable (given not all machines may have an `ib0` interface? 

If we agree to keep that `INTERFACE` variable, keeping the interface parametrized here should be fine.

Reply:
Lets keep it for now